src/main/java/uk/gov/cshr/controller/signup/SignupController.java

```diff
@@ -93,14 +99,24 @@ public String sendInvite(Model model,
         }
 
         final String email = form.getEmail();
-
-        if (inviteRepository.existsByForEmailAndStatus(email, InviteStatus.PENDING)) {
+        Optional<Invite> invite = inviteService.findByForEmailAndStatus(email, InviteStatus.PENDING);
+        if(invite.isPresent()) {
+            if (!inviteService.ifExpired(invite.get())) {
+                long timeForReReg = new Date().getTime() - invite.get().getInvitedAt().getTime();
+                if (timeForReReg < allowedTimeForReReg * 1000) {
+                    log.info("{} user given time to wait before re-registration", email);
+                    redirectAttributes.addFlashAttribute(
+                            ApplicationConstants.STATUS_ATTRIBUTE,
+                            "You have been sent an email with a link to register your account. Please check your spam or junk mail folders.\n" +
+                                    "If you have not received your link, please wait 24 hours and re-enter your details to create an account. ");
```

Comment:
24 hours should not be hardcoded in the message. Please use the property value.

Reply:
The time value i now fetched from application.yml file. So its now configurable.